README.md

```diff
@@ -1,20 +1,12 @@
 # somatic-cnvs
-Workflows for somatic copy number variant analysis
 
-## Running the Somatic CNV WDL
+### Purpose :
+Workflows for somatic copy number variant analysis.
 
-### Which WDL should you use?
+### cnv_somatic_panel_workflow :
+Builds a panel of normals (PoN) for the cnv pair workflow.
```

Comment:
Let's use "PON" (capitalized O) to be consistent with this: https://gatkforums.broadinstitute.org/gatk/discussion/11053/panel-of-normals-pon